packages/unified-backend/graphqlapp/index.js

```diff
@@ -21,36 +20,90 @@ const typeDefs = gql`
         level: Int,
         campus: String, 
         term: String,
+        breadths: [Int],
+        meeting_sections: [Meeting_Section]
+     }
+     type Meeting_Section{
+        code: String,
+        instructors: [String],
+        times: [Times],
+        size: Int,
+        enrolment: [Int]
+     }
+     type Times{
+        day: String,
+        start: Int,
+        end: Int,
+        duration: Int, 
+        location: String
+     }
+     type Subject {
+        name: String,
+        degrees: [String],
+        notes: [String],
+        programs: [Program]
+     }
+     type Program{
+         name: String,
+         level: String,
+         code: String,
+         type: String,
+         notes: [String]!,
+         courses: [Classes]
+     }
+     type Classes {
+         Int: [String]
      }
-
      type Query {
-         courses: [Course]
+         courses: [Course],
+         programs: [Subject]
```

Comment:
Why is the program query returning Subjects?

Reply:
the name that shawn set for the dataase